firmware/src/src/defines.h

```diff
@@ -18,6 +18,12 @@
 #define FW_VERSION STRINGIFY(VERSION)
 #define FW_BOARD "NANO33BLE"
 
+#if defined(DEBUG)
+#define DEFAULT_LOG_LEVEL DEBUG
```

Comment:
this is ok for the default -- the new setup allows us to dynamically update the log level -- so we can have a user turn on debug in the field if needed, etc.